lib/main.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:first_app/src/card_page.dart';
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
```

Comment:
For the sake of readabilty les change the name of the classes. https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0

Reply:
Thanks for the advice and the article. Done